test/integ/test_cursor.py

```diff
@@ -1583,3 +1583,29 @@ def test_null_connection(conn_cnx):
             status = con.get_query_status(cur.sfqid)
             assert status == QueryStatus.FAILED_WITH_ERROR
             assert con.is_an_error(status)
+
+
+@pytest.mark.skipolddriver
+def test_first_chunk_log(conn, db_parameters, caplog):
+    caplog.set_level(logging.INFO)
+    with conn() as cnx:
+        c = cnx.cursor()
+        c.execute("select 1")
+        assert "Number of results in first chunk: 1" in caplog.text
+
+        try:
+            c.execute(
+                "insert into {name}(aa) values(123456),"
+                "(98765),(65432)".format(name=db_parameters["name"])
```

Comment:
You shouldn't be using `db_parameters["name"]` instead please use the name randomizer. One example in this file: `random_string(5, "test_describe_")`